lib/src/message_listview.dart

```diff
@@ -74,6 +75,7 @@ class MessageListView extends StatefulWidget {
     this.messagePadding = const EdgeInsets.all(8.0),
     this.textBeforeImage = true,
     this.messageDecorationBuilder,
+    this.scrollPhysics = const AlwaysScrollableScrollPhysics(),
```

Comment:
this seems like a reasonable default for our use case but I figured I'd expose this (as opposed to just setting it directly below) in case we ever need to pass in anything different here

Reply:
Seems like something _they_ should have exposed as well 👍